### DISCUSSION TITLE
How to solve FAILURE: Build failed with an exception.  * What went wrong: Execution failed for task ':rat'.

### DISCUSSION THREAD

**Discussion by rich7420** [apache/gravitino#3328] [Q&A] [answered]
Hello everyone,I'm new to Gravitino. I want to contribute to this Opensource project.
My environment is Ubuntu22.04 that had installed Java with ./gradlew javaToolchains
I follow https://datastrato.ai/docs/0.5.0/how-to-build/ to build Gravitino Server.
when I use ./gradlew build or ./gradlew build -PjdkVersion=11, the problem appear.
Constractly , when I use ./gradlew build -x test , none problem appear.
![image](https://github.com/datastrato/gravitino/assets/101171023/6a431787-66ec-44eb-99ce-5ebcbf6433d2)
![image](https://github.com/datastrato/gravitino/assets/101171023/22cccb92-0c27-4180-9357-d54036d60c8a)
I searched ':rat' problem. From this [article](https://stackoverflow.com/questions/45926053/files-with-unapproved-unknown-licenses-when-running-a-gradle-project),maybe I can add failOnError=false.
but I don't know where can be added.
I would be very grateful if someone helps me.

**Comment by yuqi1129** [comment]
@rich7420 
You may need to run `./gradlew clean` and it will remove the file 'gravitino-server.out' in the screenshot that causes the failure.

**Reply by rich7420** [reply]
Thank you for your answer. I am going to try it.

**Reply by rich7420** [reply]
I tried this way but it seems not working. But I am grateful for your answer. 
![image](https://github.com/datastrato/gravitino/assets/101171023/757c5d5e-a3b1-4fef-80ee-8c635ff505b5)

**Reply by yuqi1129** [reply, marked as answer]
You need to manually remove it as `./gradlew clean` will not drop file `gravitino-server.out`.

**Reply by rich7420** [reply]
It works! Thank you for your answer!!!
![image](https://github.com/datastrato/gravitino/assets/101171023/e347c5dd-510c-4f91-a7b9-e652e1fd942c)
